src/client/components/cube-view/cube-view.tsx

```diff
@@ -41,6 +43,7 @@ export interface CubeViewState {
   showRawDataModal?: boolean;
   RawDataModalAsync?: typeof RawDataModal;
   downloadableDataset?: Dataset;
+  layout?: {left: number, right: number};
```

Comment:
I am not a huge fan of inline interfaces, could you move this into its own interface maybe call it `CubeViewLayout` ? Also I am not sure about 'left' and 'right', they sound too generic. Would a more descriptive name be better? Like `dimensionPanelWidth` and `pinboardWidth`